scripts/novaseq/demux-novaseq.batch

```diff
@@ -0,0 +1,52 @@
+#!/bin/bash
```

Comment:
Done in https://github.com/Clinical-Genomics/demultiplexing/pull/134/commits/fc5b034543a5caedeef4f89ee9abbd2c7234b016, but isn't `.batch` making it very recognizable as the job submission file? It seems the extension in itself does not do anything, I've seen examples that use `srun` for instance :)

Reply:
I might be biased because we use ".sh" in MIP, 😄. 
However, I think batch is for: "A batch file is a script file in DOS, OS/2 and Microsoft Windows."
While ".sh" id: "sh files are unix (linux) shell executables files, they are the equivalent (but much more powerful) of bat files on windows."
Since we use linux I thought ".sh" to be more appropriate